plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -3311,9 +3311,11 @@ def self.select_best_fit_node_impl(server_infos)
         # Take all nodes with > the average remaining capacity.  Make sure to take at least max(4, 20% of nodes) nodes.
         server_infos.sort_by! { |server_info| server_info.node_consumed_capacity }.reverse!
         node_consumed_capacities = server_infos.map { |server_info| server_info.node_consumed_capacity }
-        average_consumed_capacity = (node_consumed_capacities.inject(0.0) { |sum, c| sum + c } / node_consumed_capacities.length) + 1
+        average_consumed_capacity = (node_consumed_capacities.inject(0.0) { |sum, c| sum + c } / node_consumed_capacities.length)
+        # Add a little so average isn't as harsh at the low end
+        cut_off_capacity = average_consumed_capacity + (1 - average_consumed_capacity/100)
```

Comment:
The reason the +1 was in there was to remove the rounding error at the lower end.  Say you have consumed capacity of:

5,5,5,6,6,6

Which would give an average of 5.5.  The normal algorithm would leave out 2 of the 6s.

But if you have:

99, 99, 99, 98, 98, 98

You probably do want to leave off the 99s.

And the new calculation is meant to solve both.